src/panoramic/cli/connections.py

```diff
@@ -235,6 +239,31 @@ def test(cls, credentials) -> Tuple[bool, str]:
         # Create dialect specific adapter that handles connections
         adapter_cls = get_adapter_class_by_name(credentials.type)
         adapter = adapter_cls(adapter_config)  # type: ignore
+        return adapter
+
+    @classmethod
+    def execute(cls, sql: str, credentials) -> Tuple[str, Any]:
+        adapter = cls.get_connection_adapter(credentials)
+
+        with adapter.connection_named('pano'):  # type: ignore
```

Comment:
Can you elaborate on why the value `pano` is hardcoded here?

Reply:
No real reason - wanted to name it somehow when I copied the code from a dbt util example. I think we could use a blank value (that probably defaults to "main" connection).